azure/scope/cluster.go

```diff
@@ -733,6 +740,27 @@ func (s *ClusterScope) CloudProviderConfigOverrides() *infrav1.CloudProviderConf
 	return s.AzureCluster.Spec.CloudProviderConfigOverrides
 }
 
+// ExtendedLocationName returns ExtendedLocation name for the cluster.
+func (s *ClusterScope) ExtendedLocationName() string {
```

Comment:
These are simple functions, but we should add unit tests in `cluster_test.go` to exercise them and use `ExtendedLocation`.

Reply:
Has added unit tests for ExtendedLocationName() and ExtendedLocationType().